brainiak/eventseg/event.py

```diff
@@ -60,14 +60,12 @@ class EventSegment(BaseEstimator):
     n_iter: int : default 500
         Maximum number of steps to run during fitting
 
+    event_chains: ndarray with length = n_events
+        Array with unique value for each separate chain of events, each linked
+        in the order they appear in the array
+
     Attributes
     ----------
-    p_start, p_end: length n_events+1 ndarray
-        initial and final prior distributions over events
-
-    P: n_events+1 by n_events+1 ndarray
-        HMM transition matrix
-
```

Comment:
I am not sure if someone was really using these attributes, but please add a "removal" news fragment announcing their removal.

Reply:
I actually decided to revert this change and keep these attributes publicly exposed, especially since the P matrix is now more complicated and so it could help users to debug.